draft-ietf-jose-fully-specified-algorithms.xml

```diff
@@ -53,18 +53,22 @@
 	that fully specify the cryptographic operations to be performed,
 	including any curve, key derivation function (KDF), and hash functions,
 	as being "fully specified".
-	Whereas, it refers to cryptographic algorithm identifiers
+	It refers to cryptographic algorithm identifiers
 	that require additional information beyond the algorithm identifier
 	to determine the cryptographic operations to be performed
 	as being "polymorphic".
 	This specification creates fully-specified algorithm identifiers for registered
 	JSON Object Signing and Encryption (JOSE) and
 	CBOR Object Signing and Encryption (COSE)
-	polymorphic algorithm identifiers and deprecates those polymorphic algorithm identifiers,
+	polymorphic algorithm identifiers,
 	enabling applications to use only fully-specified algorithm identifiers.
+	It deprecates those polymorphic algorithm identifiers.
```

Comment:
```suggestion
	It deprecates all presently registered polymorphic algorithm identifiers.
```

Reply:
The wording "It deprecates all presently registered polymorphic algorithm identifiers." would be misinterpreted.  We don't deprecate all registered polymorphic algorithm identifiers.  We only deprecate those for which we register replacement algorithms, hence the "those".